src/main/java/edu/harvard/iq/dataverse/api/MxrdrWorkflowApi.java

```diff
@@ -0,0 +1,125 @@
+package edu.harvard.iq.dataverse.api;
+
+import static edu.harvard.iq.dataverse.workflow.execution.WorkflowContext.TriggerType.PostPublishDataset;
+
+import java.util.Optional;
+
+import javax.inject.Inject;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+import edu.harvard.iq.dataverse.engine.command.exception.NoDatasetFilesException;
+import edu.harvard.iq.dataverse.persistence.dataset.Dataset;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetRepository;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetVersion;
+import edu.harvard.iq.dataverse.persistence.user.AuthenticatedUser;
+import edu.harvard.iq.dataverse.persistence.workflow.Workflow;
+import edu.harvard.iq.dataverse.persistence.workflow.WorkflowExecution;
+import edu.harvard.iq.dataverse.workflow.WorkflowServiceBean;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowContext;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowContext.TriggerType;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowExecutionFacade;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowExecutionService;
+
+@Path("mxrdr")
+public class MxrdrWorkflowApi extends AbstractApiBean {
```

Comment:
I don't see the reason why this is not in the main dataverse repo. We don't use any `mxrdr` specific things here

Reply:
Moved to https://github.com/CeON/dataverse/pull/1327